File: vue-vben-admin/apps/web-naive/src/api/entity/SysFileDocument.ts
export interface SysFileDocument {
  name: string  | null | undefined ;    //  文件名
  path: string  | null | undefined ;    //  文件路径
  ext: string  | null | undefined ;    //  后缀
  storeType: string  | null | undefined ;    //  存储类型
  bucketName: string  | null | undefined ;    //  存储桶
  tenantId: string  | null | undefined ;    //  租户GUID
  id: string  | null | undefined ;    //  主键
  createdId: string  | null | undefined ;    //  创建人GUID
  createdTime: string  | null | undefined ;    //  创建时间
  creator: string  | null | undefined ;    //  创建人
  updatedTime: string  | null | undefined ;    //  更新时间
  updatedId: string  | null | undefined ;    //  更新人GUID
  updator: string  | null | undefined ;    //  更新人
}

File: src/SimpleApp/Simple.AdminApplication/SysMng/Entities/SysFileDocument.cs
namespace Simple.AdminApplication.SysMng.Entities;

/// <summary>
/// 文件信息
/// </summary>
public class SysFileDocument : DefaultTenantEntity
{
    /// <summary>
    /// 文件名
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// 文件路径
    /// </summary>
    public string Path { get; set; }

    /// <summary>
    /// 后缀
    /// </summary>
    public string Ext { get; set; }

    /// <summary>
    /// 存储类型
    /// </summary>
    public string StoreType { get; set; }

    /// <summary>
    /// 存储桶
    /// </summary>
    public string BucketName { get; set; }
}